src/modules/AutoInstall.rb

```diff
@@ -330,6 +337,43 @@ def pkg_gpg_check(data)
       accept ? "I" : ""
     end
 
+    # Checking for valid imported values and there is an fatal error
+    # we will stop the installation.
+    #
+    # @return [Boolean] True if the proposal is valid or the user accepted an invalid one.
+    def valid_imported_values
+      return true if @issues_list.empty?
+
+      report_settings = Report.Export
+      if @issues_list.fatal?
+        # On fatal errors, the message should be displayed and without timeout
+        level = :error
+        buttons_set = :abort
+        display_message = true
+        log_message = report_settings["errors"]["log"]
+        timeout = 0
+      else
+        # On non-fatal issues, obey report settings for warnings
+        level = :warn
+        buttons_set = :question
+        display_message = report_settings["warnings"]["show"]
```

Comment:
I did not check, but `report_settings["warnings"]` are always defined? What if they aren't?

Reply:
Good point. We have checked it. Yes, it is defined :-)